src/controllers/resume.submission.controller.ts

```diff
@@ -31,11 +33,28 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     if (isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
       const appData: ApplicationData = await getOverseasEntity(req, transactionId, overseaEntityId);
 
-      if (!appData || !Object.keys(appData).length) {
+      if (!Object.keys(appData || {}).length) {
         throw createAndLogErrorRequest(req, `Error on resuming OE - ${infoMsg}`);
       }
 
-      setWebApplicationData(req.session as Session, appData, transactionId, overseaEntityId);
+      const session = req.session as Session;
+      setWebApplicationData(session, appData, transactionId, overseaEntityId);
+
+      const resource = await getTransaction(req, transactionId);
```

Comment:
i'd rename the resource const to be 'transaction'

Reply:
I will rename it to `transactionResource`